.github/workflows/ci.yml

```diff
@@ -211,3 +212,21 @@ jobs:
         with:
           name: output
           path: _output/**
+
+      - name: Publish Artifacts to S3
+        run: make -j2 publish BRANCH_NAME=${GITHUB_REF##*/}
+        if: env.AWS_ACCESS_KEY_ID != ''
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
We have already injected the `AWS_ACCESS_KEY_ID` in the workflow environment variables (line 21). Do we still need to do again it here?

Reply:
Done. Removed the step override. We should be fine without the override.